README.md

```diff
@@ -133,6 +133,42 @@ limit is reached or no more keys are available. Uses [s3:listObjectsV2](https://
 Returns a non-ASCII key representation for an encoded s3 key. Useful to obtain the
 not-encoded key representation after calling `listObjects`.
 
+#### dy.Model({ name: String, attributes: Object, indices: Object })
+Instantiates Model.<br>
+Internally uses [dynamodb-toolbox](https://github.com/jeremydaly/dynamodb-toolbox)
+
+#### dy.Model.upsert(item: Object, opts: Object)
+Creates entry if key does not exist. Otherwise updates the item.<br>
+Options include:
+- `conditions` (Object|Array): Conditions that must be met for operation to succeed.
+Internally uses [update](https://github.com/jeremydaly/dynamodb-toolbox#updatekey-options-parameters)
+
+#### dy.Model.update(item: Object, opts: Object)
+Edits an existing item's attributes. Can only update an item if it exists.<br>
+Options include:
+- `returnValues` (String): Return item attributes as they appeared before or after the update.
+- `conditions` (Object|Array): Conditions that must be met for operation to succeed.
+Internally uses [update](https://github.com/jeremydaly/dynamodb-toolbox#updatekey-options-parameters)
+
+#### dy.Model.getItemOrNull(key: String, opts: Object)
+Returns entry or null if not found.<br>
+Options include:
+- `toReturn` (Array): Fields to return.
+Internally uses [get](https://github.com/jeremydaly/dynamodb-toolbox#getkey-options-parameters)
+
+#### dy.Model.query(key: String, opts: Object)
+Pages through table based on primary key values.<br>
```

Comment:
```suggestion
Pages through table based on primary key values.<br>
```
Is that true? I think it's whatever index you pass and if no index is passed it uses the primary index